contracts/governance/FeeSharingCollectorProxy/FeeSharingCollector.sol

```diff
@@ -163,30 +162,28 @@ contract FeeSharingLogic is SafeMath96, IFeeSharingProxy, Ownable, FeeSharingPro
                 IConverterAMM(_converters[i]).withdrawFees(address(this));
 
             if (wrbtcAmountWithdrawn > 0) {
-                /// @dev TODO can be also used - function addLiquidity(IERC20Token _reserveToken, uint256 _amount, uint256 _minReturn)
-                IERC20(wRBTCAddress).approve(loanPoolToken, wrbtcAmountWithdrawn);
-                uint256 poolTokenAmount =
-                    ILoanToken(loanPoolToken).mint(address(this), wrbtcAmountWithdrawn);
+                // unwrap wrbtc to rbtc, and hold the rbtc
+                wRBTCToken.withdraw(wrbtcAmountWithdrawn);
 
                 /// @notice Update unprocessed amount of tokens
                 uint96 amount96 =
                     safe96(
-                        poolTokenAmount,
-                        "FeeSharingProxy::withdrawFees: pool token amount exceeds 96 bits"
+                        wrbtcAmountWithdrawn,
+                        "FeeSharingCollectorProxy::withdrawFeesAMM: wrbtc token amount exceeds 96 bits"
                     );
 
                 totalPoolTokenAmount = add96(
                     totalPoolTokenAmount,
                     amount96,
-                    "FeeSharingProxy::withdrawFees: total pool token amount exceeds 96 bits"
+                    "FeeSharingCollectorProxy::withdrawFeesAMM: total wrbtc token amount exceeds 96 bits"
```

Comment:
also on all of these error messages it's strange to add "proxy".

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/482/commits/61d88fbfe3ceb4b74fa8b612479c397faca5c290)